mrp_subcontracting_picking/README.rst

```diff
@@ -0,0 +1,31 @@
+========================================
```

Comment:
This README has incorrect syntax and it's not complete without description

Reply:
Thank you for mentioning this! Fixed this